wordpress/wp-content/themes/headless/functions.php

```diff
@@ -7,24 +7,30 @@
 // Load WP Config files
 //
 
-$preview_domain = '';
+$headless_domain = '';
```

Comment:
I think this block can be simplified by having the variables up top, so the logic isn't intertwined with things that have to change.